### PR TITLE
Namespaced analytics update

### PR DESCRIPTION
**Pull request by jbarnicle**

### CODE REVIEW

File: atf_eregs/settings/dev.py [hunk 1]
```diff
@@ -3,6 +3,18 @@
...
+# Analytics settings
+ANALYTICS = {
+    'GOOGLE': {
+       'GTM_SITE_ID': '',
+       'GA_SITE_ID': '',
+    },
+    'DAP': {
+        'AGENCY': 'DOJ',
+        'SUBAGENCY': 'ATF',
+    },
```
Comment (cmc333333) [inline atf_eregs/settings/dev.py]: If we don't want to send GA information, we probably don't want to send DAP info either, hey?

**Comment by cmc333333** [comment]
Would you mind deleting the packaged `Universal-Federated-Analytics-Min.js` file as well?

We can ignore CI failure until #108 is merged

File: atf_eregs/settings/dev.py [hunk 2]
```diff
@@ -3,6 +3,18 @@
...
+ANALYTICS = {
+    'GOOGLE': {
+       'GTM_SITE_ID': '',
+       'GA_SITE_ID': '',
+    },
+    'DAP': {
+        'AGENCY': '',
+        'SUBAGENCY': '',
+    },
+}
```
Comment (cmc333333) [inline atf_eregs/settings/dev.py]: Is this setting needed anymore? It should be imported from regulations-site automatically, hey?

Comment (jbarnicle) [inline atf_eregs/settings/dev.py]: Only for testing purposes.  Happy to remove if you don't think it should be there.

**Comment by cmc333333** [comment]
:+1:

**Pull request merged**